tracardi/process_engine/action/v1/connectors/api_call/model/configuration.py

```diff
@@ -10,63 +10,62 @@
 from tracardi.process_engine.tql.utils.dictonary import flatten
 from pydantic import BaseModel, validator
 from tracardi.domain.named_entity import NamedEntity
-from pydantic import AnyHttpUrl, BaseModel, validator
 from tracardi.service.plugin.domain.config import PluginConfig
 
 
 class Method(str, Enum):
-    post = "post"
-    get = "get"
-    put = "put"
-    delete = 'delete'
+	post = "post"
+	get = "get"
+	put = "put"
+	delete = 'delete'
 
 
 class Content(BaseModel):
-    type: str
-    content: Optional[str]
-
-    @validator("content")
-    def validate_content(cls, value, values):
-        if 'type' in values and values['type'] == 'application/json':
-            try:
-                # Try parsing JSON
-                json.loads(value)
-            except JSONDecodeError as e:
-                raise ValueError(str(e))
-        return value
-
-    def load_body_as_dict(self):
-        return json.loads(self.content)
+	type: str
+	content: Optional[str]
+	
+	@validator("content")
+	def validate_content(cls, value, values):
+		if 'type' in values and values['type'] == 'application/json':
+			try:
+				# Try parsing JSON
+				json.loads(value)
+			except JSONDecodeError as e:
+				raise ValueError(str(e))
+		return value
+	
+	def load_body_as_dict(self):
+		return json.loads(self.content)
 
 
 class RemoteCallConfiguration(PluginConfig):
-    source: NamedEntity
```

Comment:
Could you check why this file is reformatted. Maybe your setting are not the same as ours?

Reply:
If you could keep it the same then it would be great.

Reply:
i had it set to spaces for continuation indent 8 instead of 4 for some reason